docs/01-app/03-api-reference/04-functions/setTagStale.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: setTagStale
+description: API Reference for the setTagStale function.
+---
+
+`setTagStale` allows you to mark [cached data](/docs/app/guides/caching) as stale for specific cache tags without immediately invalidating it.
+
+## Usage
+
+`setTagStale` can be called in Server Functions and Route Handlers.
+
+`setTagStale` cannot be called in Client Components or Middleware, as it only works in server environments.
+
+> **Good to know**: Unlike [`revalidateTag`](/docs/app/api-reference/functions/revalidateTag), `setTagStale` marks data as stale but preserves the existing timestamp under an `expire` field in the tag manifest. This allows for more granular cache control where data can be marked stale without immediate invalidation.
```

Comment:
Should update this to clarify what stale means without discussing manifest changes